src/styles/index.scss

```diff
@@ -1,3 +1,129 @@
-body {
+.page {
   margin: 0;
+  box-sizing: border-box;
+  width: 200px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+
+  background-position: center;
+  background-size: cover;
+  margin: 33 20 0;
+}
+
+.card__title {
+  display: block;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40 17 0;
+  color: rgba(6, 11, 53, 1);
+  height: 36px;
+}
+
+.card__code {
+  display: block;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: rgba(97, 96, 112, 1);
+  margin: 4 0 0 17;
```

Comment:
The margin shorthand property in `.card__code` is missing units for its values, similar to the `.card__img` and `.card__title` classes. It should be `margin: 4px 0 0 17px;`.